Makefile

```diff
@@ -18,8 +18,4 @@ coverage:
 	XDEBUG_MODE=coverage vendor/bin/phpunit --coverage-html=coverage
 
 run-all-examples:
-	for file in ./examples/*.php; do \
-		echo "Running $$file..."; \
-		php $$file; \
-		echo ""; \
-	done
+	./example
```

Comment:
Didn't now 😃

Reply:
Ah it's a command now 😅

Reply:
was just annoyed of waiting and was keen building that :D